build_scripts/windows/scripts/build.cmd

```diff
@@ -12,7 +12,7 @@ if "%CLI_VERSION%"=="" (
 set PYTHON_VERSION=3.6.6
 
 set WIX_DOWNLOAD_URL="https://azurecliprod.blob.core.windows.net/msi/wix310-binaries-mirror.zip"
-set PYTHON_DOWNLOAD_URL="https://azurecliprod.blob.core.windows.net/util/Python366-32.zip"
+set PYTHON_DOWNLOAD_URL="https://azurecliprod.blob.core.windows.net/util/Python368-64.zip"
```

Comment:
why upgrade the embedded python version to 3.8?

Reply:
No 3.8, but 3.6.8 with security fixes.